controllers/helmchart_controller_test.go

```diff
@@ -546,10 +546,10 @@ var _ = Describe("HelmChartReconciler", func() {
 			}, timeout, interval).Should(BeTrue())
 
 			By("Committing a new version in the chart metadata")
-			f, err := fs.OpenFile(fs.Join(chartDir, chartutil.ChartfileName), os.O_RDWR, os.FileMode(0600))
+			f, err := fs.OpenFile(fs.Join(chartDir, "helmchartwithdeps", chartutil.ChartfileName), os.O_RDWR, os.FileMode(0600))
 			Expect(err).NotTo(HaveOccurred())
 
-			b := make([]byte, 1024)
+			b := make([]byte, 2048)
```

Comment:
Had to increase the allocation size due to an increase in the YAML size after adding dependencies.